diff --git a/src/xmlpatterns/xmlpatterns.pro b/src/xmlpatterns/xmlpatterns.pro
index c1960c7..8584cb1 100644
--- a/src/xmlpatterns/xmlpatterns.pro
+++ b/src/xmlpatterns/xmlpatterns.pro
@@ -10,7 +10,10 @@ QT = core-private network
 
 DEFINES += QT_BUILD_XMLPATTERNS_LIB QT_NO_USING_NAMESPACE
 win32-msvc*|win32-icc:QMAKE_LFLAGS += /BASE:0x61000000
-unix|win32-g++*:QMAKE_PKGCONFIG_REQUIRES = QtCore QtNetwork
+unix|win32-g++* {
+    QMAKE_PKGCONFIG_REQUIRES = QtCore QtNetwork
+    CONFIG += create_prl create_pc
+}
 load(qt_module_config)
 
 HEADERS += qtxmlpatternsversion.h
