diff --git a/qtxmlpatterns.pro b/qtxmlpatterns.pro
index f3bd6fe..f593575 100644
--- a/qtxmlpatterns.pro
+++ b/qtxmlpatterns.pro
@@ -10,6 +10,7 @@ module_qtxmlpatterns_tools.depends = module_qtxmlpatterns_src
 module_qtxmlpatterns_examples.subdir = examples/xmlpatterns
 module_qtxmlpatterns_examples.target = module-qtxmlpatterns-examples
 module_qtxmlpatterns_examples.depends = module_qtxmlpatterns_src
+module_qtxmlpatterns_examples.CONFIG = no_default_install
 
 module_qtxmlpatterns_tests.subdir = tests
 module_qtxmlpatterns_tests.target = module-qtxmlpatterns-tests
